src/runtime/tests/refiner-test.ts

```diff
@@ -8,21 +8,25 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import {Refiner} from '../refiner.js';
+import {Range, Segment, Refinement, BinaryExpression, UnaryExpression} from '../refiner.js';
 import {parse} from '../../gen/runtime/manifest-parser.js';
 import {assert} from '../../platform/chai-web.js';
-
+import {Manifest} from '../manifest.js';
+import {Entity, EntityClass} from '../entity.js';
+import {Schema} from '../schema.js';
+import {Flags} from '../flags.js';
 describe('refiner', () => {
     it('Refines data given an expression.', () => {
         const manifestAst = parse(`
             particle Foo
                 input: reads Something {num: Number [ (num/-3 < 0 and num*num == 36) or num == 0 ] }
         `);
-        const ref = manifestAst[0].args[0].type.fields[0].type.refinement;
+        const typeData = {'num': 'Number'};
+        const ref = Refinement.fromAst(manifestAst[0].args[0].type.fields[0].type.refinement, typeData);
```

Comment:
Consider making a helper function to extract the refinement field; `manifestAst[0].args[0].type.fields[0].type.refinement` is repeated a lot through here.